src/data-types/arrays/adding-removing-elements.js

```diff
@@ -10,25 +10,25 @@ const fruits = ['Apple', 'Orange', 'Pear']
 // TODO: write code to pass the tests
 
 // Edit this code to add 'Fred' to the names array
-names.push(undefined)
+names.push('Fred')
 
 // Edit this code to add 4 to the end of the numbers array
-numbers.push(NaN)
+numbers.push(4)
 
 // Edit this code to add 'Rio' to the start of the cities array
-cities.unshift(undefined)
+cities.unshift('Rio')
 
 // Use an array method to remove the first item from colours
-colours
+colours.shift('Red')
```

Comment:
shift() can be called without an argument (value) and that will remove the first item, see the documentation here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/shift